src/middlewared/middlewared/api/v25_04_0/ftp.py

```diff
@@ -0,0 +1,86 @@
+import logging
```

Comment:
Unused logger object and unused logging module import

Reply:
oof... and why we have reviews.